src/api/public/apidocs-new/paths/source_project_cmd_createmaintenanceincident.yaml

```
@@ -0,0 +1,50 @@
+post:
+  summary: Create a single maintenance incident project
```

```suggestion
  summary: Create a single maintenance incident project.
```
Missing dot at the end of the sentence.

---

It's updated.